src/main/java/sirius/pasta/noodle/macros/Base64ResourceMacro.java

```diff
@@ -99,7 +100,12 @@ public String eval(FunctionCall call) {
             path = path.substring(1, path.length() - 1);
         }
 
-        return encodeResource(path);
+        String code = encodeResource(path);
+        if (Strings.isEmpty(code)) {
+            throw new IllegalArgumentException("Unknown resource: " + path);
```

Comment:
Alternatively, we could also throw an exception in `encodeResource(String)` above, instead of returning an empty string. That would also affect the Tagliatelle part, though, and not just the SASS part. Opinions?